content/kubernetes-platform-services/dashboard/kubernetes-ui-dashboard.md

```diff
@@ -2,17 +2,40 @@
 title: "Kubernetes UI Dashboard"
 description: ""
 ---
-# Synopsis
-
-Explain what it does
 
 # Dependencies
 
 * None
 
-# Example
+# Install
+
+You can install `kubernetes-dashboard` in a few different ways, but we recomend to use the [Master Helmfile](https://github.com/cloudposse/geodesic/blob/master/rootfs/conf/kops/helmfile.yaml).
+
+Kubernetes dashboard required `heapster` to show `cpu` and `memory` charts on dashboard.
+You can skip install `heapster` if there is no need in resources chart.
+
+## Install with Master Helmfile
+
+Run following command
+```
+helmfile -f /conf/kops/helmfile.yaml --selector namespace=kube-system,chart=kubernetes-dashboard sync
+helmfile -f /conf/kops/helmfile.yaml --selector namespace=kube-system,chart=heapster sync
+```
+This environment variables can be useful for configure:
```

Comment:
These environment variables are used to configure:

Reply:
Done